darwin/importer/formats/labelbox_schemas.py

```diff
@@ -35,6 +35,15 @@
     "items": point,
 }
 
+polyline = {
+    "$id": "https://darwin.v7labs.com/schemas/polyline",
+    "description": "Schema of a Polyline",
+    "title": "Polyline",
+    "default": [{"x": 1.2, "y": 2.5}, {"x": 2.5, "y": 3.6}, {"x": 1.2, "y": 2.5}],
+    "examples": [[{"x": 1.2, "y": 2.5}, {"x": 2.5, "y": 3.6}, {"x": 1.2, "y": 2.5}], []],
```

Comment:
Is this way of specifying floats (`1.2` or `2.5`) expected?

Reply:
resolved over slack